2022/en/src/K06-broken-authentication.md

```diff
@@ -1,6 +1,7 @@
 ## Overview
 Authentication in Kubernetes takes on my many forms and is extremely flexible. This emphasis on being highly configurable makes Kubernetes work in a number of different environments but also presents challenges when it comes to cluster and cloud security posture.
 
+![Broken Authentication - Illustration](/assets/images/K06-2022.gif)
```

Comment:
Do you mean OIDC as opposed to ODIC?

Reply:
Open ID Connect. This is a typo that needs fixing.

Reply:
OIDC. Fixed.